app/scripts/views/confirm.js

```diff
@@ -55,9 +55,13 @@ define(function (require, exports, module) {
         canGoBack: isSignIn && this.canGoBack(),
         email: email,
         gmailLink: this.getGmailUrl(email),
+        hotmailLink: this.getHotmailUrl(),
         isOpenGmailButtonVisible: this.isOpenGmailButtonVisible(email),
+        isOpenHotmailButtonVisible: this.isOpenHotmailButtonVisible(email),
+        isOpenYahooButtonVisible: this.isOpenYahooButtonVisible(email),
         isSignIn: isSignIn,
-        isSignUp: isSignUp
+        isSignUp: isSignUp,
+        yahooLink: this.getYahooUrl()
```

Comment:
this would have `webmailLink` that calls `this.getWebmailLink` that determines the link to show based on the `email`